client/src/pages/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,97 @@
+import { useContext } from 'react';
+import FullCalendar, { EventClickArg, EventDropArg, EventHoveringArg } from '@fullcalendar/react'; // must go before plugins
+import dayGridPlugin from '@fullcalendar/daygrid'; // a plugin!
+import timeGridPlugin from '@fullcalendar/timegrid';
+import interactionPlugin, { EventResizeDoneArg } from '@fullcalendar/interaction';
+
+import { BoardContext } from '../../context/useBoardContext';
+import { ICard } from '../../interface';
+
+const Calendar = (): JSX.Element => {
+  const [board, setBoard] = useContext(BoardContext);
+
+  const handleClick = (info: EventClickArg) => {
+    // you need to render the card details popup here
+    alert(`${info.event.title} was clicked`);
+  };
+
+  const handleEventDrop = (info: EventDropArg) => {
+    const {
+      id,
+      title,
+      extendedProps: { description },
+      backgroundColor,
+      start,
+      end,
+    } = info.event.toPlainObject();
+
+    const newBoard = { ...board };
+    newBoard.cards[id] = {
+      id,
+      title,
+      description,
+      color: backgroundColor,
+      start,
+      end,
+    };
+
+    setBoard(newBoard);
+  };
+
+  const handleEventResize = (info: EventResizeDoneArg) => {
+    const {
+      id,
+      title,
+      extendedProps: { description },
+      backgroundColor,
+      start,
+      end,
+    } = info.event.toPlainObject();
+
+    const newBoard = { ...board };
+    newBoard.cards[id] = {
+      id,
+      title,
+      description,
+      color: backgroundColor,
+      start,
+      end,
+    };
+
+    setBoard(newBoard);
+  };
+
+  const handleMouseEnter = (info: EventHoveringArg) => {
+    // show something fancy about the current card
+  };
```

Comment:
Not yet implemented?

Reply:
It is a fancy feature, it wasn't part of the ticket I however thought it would be a good add but I didn't want to spend much time on it. I'd set it up once the main tasks are done.